deepchecks/tabular/dataset.py

```diff
@@ -1061,6 +1065,172 @@ def datasets_share_date(cls, *datasets: 'Dataset') -> bool:
 
         return True
 
+    @property
+    def _dataset_description(self) -> t.Tuple[
+        t.List[t.Union[str, int]],
+        pd.DataFrame,
+        pd.DataFrame
+    ]:
+        data = self.data
+        datetime_name = self.datetime_name
+        index_name = self.index_name
+        label_name = self.label_name
+        features = self.features
+        categorical_features = self.cat_features
+        numerical_features = self.numerical_features
+
+        columns = features[::]
+
+        if label_name is not None:
+            columns = [*columns, label_name]
+
+        is_index_name_str = isinstance(index_name, str)
+        is_index_name_int = isinstance(index_name, int)
+        is_multiindex = isinstance(data.index, pd.MultiIndex)
+
+        unamed_index_levels = {
+            level
+            for level, name in enumerate(data.index.names)
+            if name is None
+        }
+
+        if index_name in data.columns:
+            columns = [index_name, *columns]
+
```

Comment:
don't have blank lines between if/else

Reply:
why?
I think code becomes more readable when there is one line between multiline `if/elif/else` branches

Reply:
I think it's all part of the same code block, and the blank lines separates it and make it harder to read for me

Reply:
fixed